src/test/unit/services/sample/fixed_param_test.cpp

```diff
@@ -153,51 +158,5 @@ TEST_F(ServicesSamplesFixedParam, parameter_checks) {
   EXPECT_DOUBLE_EQ(0.0, parameter_values.back()[2])
     << "final memory_writer should be (0, 0)";
   EXPECT_EQ(return_code, 0);
-
 }
 
-TEST_F(ServicesSamplesFixedParam, output_regression) {
```

Comment:
True, but then it can be slightly ambiguous as to which particular failure caused the entire test to fail.  Typically we have avoided monolithic tests and kept them split up roughly by the functionality being tested even with the overhead of repeated instantiations.  It's not a deal-breaker, but I would prefer to maintain the current convention if possible.